.uncrustify

```diff
@@ -240,7 +240,7 @@ nl_assign_brace                 = remove
 nl_assign_leave_one_liners      = true
 nl_assign_square                = ignore
 nl_before_access_spec           = 0
-nl_before_block_comment         = 2
+nl_before_block_comment         = 0
```

Comment:
Is there a particular reason why you decided to make this change?

Reply:
I wanted to put a multiline comment in the middle of a block, and `uncrustify` treats any multiline comment as a "block" comment, even if it doesn't start with `/*` or `/**` on a line by itself.

Reply:
To be clear, this doesn't prevent adding newlines there, it just doesn't force them to be present. Also, this doesn't affect empty lines between functions.

Reply:
Sounds good to me. Thanks for the explanation